.github/workflows/links-verify.yml

```diff
@@ -46,12 +46,21 @@ jobs:
         -e 'https://api.*.crowdstrike.com.*' \
         -e 'https://dehashed.com.*' \
         -e 'https://secure.aadcdn.microsoftonline-p.com.*' \
-        https://xsoar.pan.dev > muffet.out 2>&1 && echo "muffet completed successfully" && exit 0 || echo "muffet failed but still need to verify output file..."
-        grep -v -E '^\s+(403|401|x509|lookup|parse|timeout|dialing)' muffet.out > muffet-clean.out
+        -e 'https://blog.demisto.com.*' \
```

Comment:
remind me what are these links for?

Reply:
The links are ignored by muffet (link verifier). For some reason muffet started having problems with demisto.com links when they started redirecting to panw site.